pysassc.py

```diff
@@ -86,12 +122,42 @@ def main(argv=sys.argv, stdout=sys.stdout, stderr=sys.stderr):
             output_styles + '. [default: %default]'
         ),
     )
+    parser.add_option(
+        '-i', '--indented-syntax', dest='indented', default=False,
+        help='Treat data from stdin as sass code (versus scss).',
+    )
     parser.add_option(
         '-m', '-g', '--sourcemap', dest='source_map',
         action='store_true', default=False,
         help='Emit source map.  Requires the second argument '
              '(output css filename).',
     )
+    parser.add_option(
+        '--sourcemap-file', dest='source_map_file', metavar='FILE',
+        action='store', default=None,
+        help='Output file for source map. If omitted, source map is based on '
+             'the output css filename',
+    )
+    parser.add_option(
+        '--sourcemap-contents', dest='source_map_contents',
+        action='store_true', default=False,
```

Comment:
can omit `default=False` for `store_true` (it is the default)

Reply:
(same for the others)

Reply:
For whatever reason this script is using `optparse`, not `argparse`, and it seems that the `store_true` action in optparse doesn't set the default to `False`

Reply:
urghhhh right, I haven't fixed this yet -- ignore this plus the other comment 😭

I haven't gone through and fixed these since we stopped supporting python2.6